### PR TITLE
Correct aria label for breadcrumbs

### LINKED ISSUE #5269: [A11Y] Fixed: Issues and Confusion with "You are here" screen reader heading for breadcrumbs.
## Issues and Confusion with "You are here" screen reader heading for breadcrumbs

**Description of Issue**: Breadcrumb navigation aria-labelledby name of "you are here" is confusing to screen reader users. They hear "You are here navigation" which sounds strange out of context.
**URL**:  https://pr3750-llnmetezlaix5oe10ddq5m3nya7zdz2b.tugboat.qa/exerci
**Testing Environment/Browser**: Windows Lenovo PC, Windows 10 Pro, Chrome Browser v. 94.0.4606.61, JAWS 2020.12.13
**[Severity](https://github.com/backdrop/backdrop-issues/issues/5244#issuecomment-931739260)**: P3
**Solution**: 

- in `core/includes/theme.inc`, line 2021
  1. Add an `aria-label="Website Orientation"` attribute to the `<nav>` element.
  1. Remove the `h2` containing `You are here`.
- in `core/modules/system/system.module`:, line 3274
  1. Add an `aria-current="page"` attribute to the `'<span>'` element for the current page.

A more intuitive aria-label might be: "Website Orientation." So a screen reader user would hear "Website orientation navigation"

For the breadcrumb that indicates the current page, the following ARIA attribute should be added: aria-current="page"

Code recommendation:
```html
<nav class="breadcrumb" aria-label="Website Orientation">
  <ol>
       <li><a href="/"> Home </a></li>
        ......
       <li><span aria-current="page"> Foo </span></li>
  </ol>
</nav>
```

## Documentation
* https://docs.backdropcms.org/api/backdrop/core%21includes%21theme.inc/function/theme_breadcrumb/1 
* https://docs.backdropcms.org/api/backdrop/core%21modules%21system%21system.module/function/system_breadcrumb_block/1

## Screenshot:
image

### PR DESCRIPTION
**Pull request by developer-diganta**
Fixes: https://github.com/backdrop/backdrop-issues/issues/5269
Added the `aria-label="Website Orientation"` attribute to the `<nav>` element in `core/includes/theme.inc.`
Added the aria-current for current page in `core/modules/system/system.module`.

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by klonos** [comment]
@backdrop/core-committers can one of you please approve this first-time contributor, so that the GHA tests run for this PR? Thanks.

File: core/includes/theme.inc [hunk 1]
```diff
@@ -2022,10 +2022,7 @@ function theme_breadcrumb($variables) {
   $breadcrumb = $variables['breadcrumb'];
   $output = '';
   if (!empty($breadcrumb)) {
-    $output .= '<nav class="breadcrumb">';
-    // Provide a navigational heading to give context for breadcrumb links to
-    // screen-reader users. Make the heading invisible with .element-invisible.
-    $output .= '<h2 class="element-invisible">' . t('You are here') . '</h2>';
+    $output .= '<nav class="breadcrumb" aria-label="Website Orientation">';
```
Comment (indigoxela) [inline core/includes/theme.inc]: "Website Orientation" should be wrapped in t() - otherwise it's not translatable. Same in both theme templates.

Comment (developer-diganta) [reply core/includes/theme.inc]: @indigoxela  Should I wrap the aria label text in t()? Or should I separately put Website Orientation outside of the nav element?

Comment (stpaultim) [reply core/includes/theme.inc]: @developer-diganta - I didn't know the answer, but I basically just did a search for `aria-label` in the code base to see how it is done elsewhere. There are lots of examples. Based upon those examples. This one looks particularly helpful:

image

File: core/modules/system/system.module [hunk 2]
```diff
@@ -3335,7 +3335,7 @@ function system_header_block($settings) {
 function system_breadcrumb_block($settings = array()) {
   $crumbs = backdrop_get_breadcrumb();
   if (isset($settings['current']) && $settings['current'] == TRUE) {
-    $crumbs[] = '<span>' . backdrop_get_title() . '</span>';
+    $crumbs[] = '<span aria-current="' . t('page') . "'>' . backdrop_get_title() . '</span>';
```
Comment (indigoxela) [inline core/modules/system/system.module]: This "page" shouldn't be translated, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current for some details for aria-current and the possible values.

Comment (developer-diganta) [reply core/modules/system/system.module]: @indigoxela I have removed translation for the page. Can you please check the changes?

**Comment by jenlampton** [comment]
closing / opening for new sandbox.

**Comment by laryn** [comment]
I accidentally hit return and my squash and merge happened before I was quite ready, but this has been merged for 1.x and 1.22.x.

Thanks @developer-diganta, @jenlampton, @stpaultim, @indigoxela and @klonos!

**Pull request merged**